mmengine/fileio/backends/oss_backend.py

```diff
@@ -0,0 +1,456 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import os.path as osp
+import re
+from pathlib import Path
+from typing import Iterator, Optional, Tuple, Union
+
+from .base import BaseStorageBackend
+
+
+class OSSBackend(BaseStorageBackend):
+    """OSS storage backend for reading (writing) from (to) OSS.
+
+    OSSBackend supports reading and writing data to multiple buckets
+    from Endpoint. If the file path contains the bucket name,
+    OSSBackend will read data from the specified buckets
+    or write data to them. Otherwise, OSSBackend will occur an error.
```

Comment:
```suggestion
    or write data to them. Otherwise, OSSBackend will occur an error.
    
    Please refer to https://help.aliyun.com/document_detail/85288.html
    to install the required packages of OSSBackend.
```